storybook/qmlTests/tests/tst_TokenSelectorView.qml

```diff
@@ -0,0 +1,149 @@
+import QtQuick 2.15
+import QtTest 1.15
+
+import Models 1.0
+
+import AppLayouts.Wallet.views 1.0
+import AppLayouts.Wallet.stores 1.0
+import AppLayouts.Wallet.adaptors 1.0
+
+import shared.stores 1.0
+
+Item {
+    id: root
+    width: 600
+    height: 400
+
+    QtObject {
+        id: d
+
+        readonly property var flatNetworks: NetworksModel.flatNetworks
+        readonly property var currencyStore: CurrenciesStore {}
+        readonly property var assetsStore: WalletAssetsStore {
+            id: thisWalletAssetStore
+            walletTokensStore: TokensStore {
+                plainTokensBySymbolModel: TokensBySymbolModel {}
+            }
+            readonly property var baseGroupedAccountAssetModel: GroupedAccountsAssetsModel {}
+            assetsWithFilteredBalances: thisWalletAssetStore.groupedAccountsAssetsModel
+        }
+
+        readonly property var adaptor: TokenSelectorViewAdaptor {
+            assetsModel: d.assetsStore.groupedAccountAssetsModel
+            flatNetworksModel: d.flatNetworks
+            currentCurrency: d.currencyStore.currentCurrency
+        }
+    }
+
+    Component {
+        id: componentUnderTest
+        TokenSelectorView {
+            anchors.fill: parent
+
+            model: d.adaptor.outputAssetsModel
```

Comment:
The goal of unit tests is to test components in isolation. It can be easily achieved once adaptor is decoupled from the view. Instead of using adaptor here we can use just plain `ListModel` here:
```
model: ListModel { /* test data here */ }
```

Similar rules regarding isolation should be followed in storybook.

Reply:
This is just a convenience, the view is now properly separated from the adaptor; I'm just using its result as the view's model